diagnostic.md

```diff
@@ -14,7 +17,8 @@ list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
 <!-- your answer starts here -->
-
+$('#li').show();
+$('OL').find('#li');
```

Comment:
This will select `<ol>` elements and then find all (there can only be one in this case) descendants of each `<ol>` element with an `id` of `li`.